test/game_team_collection_test.rb

```diff
@@ -15,6 +15,6 @@ def test_it_exists
   def test_it_has_list_of_game_teams
     assert_instance_of Array, @game_team_collection.game_team_list
     assert_instance_of GameTeam, @game_team_collection.game_team_list.first
-    assert_equal 200, @game_team_collection.game_team_list.length
+    assert_equal 217, @game_team_collection.game_team_list.length
   end
```

Comment:
we added more files to the truncated file, so changed the test to match the truncated file